person_capture/gui_app.py

```
@@ -9094,8 +9203,34 @@ def _update_label(image: QtGui.QImage) -> None:
+            except Exception:
+                hdr_stale = True
+            if not ctx_ok or hdr_stale:
+                log.debug("SDR preview taking over preview_stack (HDR passthrough stale/dead)")
+                self._hdr_passthrough_enabled = False
+                hdr_passthrough_enabled = False
+                # Propagate to worker: disable HDR passthrough and clean up reader
+                worker = getattr(self, "_worker", None)
+                if worker is not None:
+                    try:
+                        worker.disable_hdr_passthrough()
+                    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)